src/common.js

```diff
@@ -201,6 +207,7 @@ exports.expand = expand;
 
 // Normalizes _unlinkSync() across platforms to match Unix behavior, i.e.
 // file can be unlinked even if it's read-only, see https://github.com/joyent/node/issues/3006
+// XXX: I'm not really sure why this is here or if it's still needed. I think this probobly isn't going to be a part of plugin.utils.
 function unlinkSync(file) {
```

Comment:
Would you be OK if we moved this into `src/rm.js`, and had everything else use `rm`?